js/modules/loki_public_chat_api.js

```diff
@@ -312,8 +347,11 @@ class LokiPublicChannelAPI {
       log.info(`deleted ${serverId} on ${this.baseChannelUrl}`);
       return true;
     }
+    // fire an alert
     log.warn(`failed to delete ${serverId} on ${this.baseChannelUrl}`);
-    return false;
+    throw new textsecure.PublicChatError(
```

Comment:
Where we are calling this function in `js/views/conversation_view.js` line ~1300 we aren't expecting this to throw
So we either need to catch it there and stop returning a bool, or revert this change and keep the bool as the success/fail flag

Reply:
will remove for now and re-address as parameter later